src/components/panels/WindowContainer.tsx

```diff
@@ -1,12 +1,14 @@
-import { ContentPanel, TabData } from './ContentPanel';
+import { ContentPanel, PanelData } from './ContentPanel';
 import React, { FunctionComponent } from 'react';
 
-import { TabButtonComponent } from './TabButtonComponent';
+import { PanelIconComponent } from './PanelIconComponent';
 import goodboyLogo from '../../assets/goodboy-logo.png';
 import styled from 'styled-components';
 
-export interface MainPanelProps
+export interface WindowContainerData
 {
+    /** container default position */
+    defaultContainerPosition?: { x: string , y: string };
```

Comment:
We should probably create a utils file for common types like this. That way we can have an `XY` like in odie

Reply:
thats a good shout